pyproject.toml

```diff
@@ -1,80 +1,64 @@
 [project]
-name = "connecpy"
-version = "2.3.0"
-description = "Server library and client library for the Connect Protocol"
-authors = [{ name = "Connecpy authors" }]
+name = "connect-python"
+version = "0.5.0"
+description = "Server and client runtime library for Connect RPC"
+maintainers = [
+    { name = "Anuraag Agrawal", email = "anuraaga@gmail.com" },
+    { name = "Spencer Nelson", email = "spencer@firetiger.com" },
+    { name = "Stefan VanBuren", email = "svanburen@buf.build" },
+    { name = "Yasushi Itoh", email = "i2y@todo" },
```

Comment:
@i2y I wasn't sure if you have a preferred email for OSS, please suggest one

Reply:
hmm, then please use: i2y.may.roku@gmail.com